nexus/db-queries/src/db/datastore/rack.rs

```diff
@@ -629,6 +641,46 @@ impl DataStore {
                         })?;
                     }
 
+                    // Insert the RSS-generated blueprint.
+                    Self::blueprint_insert_on_connection(
+                        &conn,
+                        opctx,
+                        &blueprint,
+                        ).await
```

Comment:
Huh, interesting indentation choices by rustfmt

Reply:
Found it. This line went past 80 columns in an unbreakable way, so we hit https://github.com/rust-lang/rustfmt/issues/3863 and rustfmt gave up on everything around:

```
                       <Zpool as DatastoreCollection<Dataset>>::insert_resource(
```

`Zpool` only implements `DatastoreCollection` once, so I removed the cast and rustfmt did its thing in aac78b0b2